test/index-spec.js

```diff
@@ -0,0 +1,114 @@
+const should = require('should');
+const {
+  validCert,
+  validKey,
+  validBundle,
+  validBundleCert,
+  nonMatchingKey
+} = require('./__fixtures__/valid-ssl');
```

Comment:
I think you should do some tests with things that *look* like keys and certs as well, but cause the pem stuff to explode (I think your code will handle it fine, but I love tests)